routers/api/v2/users.go

```diff
@@ -2,13 +2,13 @@ package v2
 
 import (
 	"encoding/json"
-	"fmt"
 	"github.com/gin-gonic/gin"
 	"github.com/pkg/errors"
 	"github.com/unicsmcr/hs_auth/authorization/v2/common"
 	"github.com/unicsmcr/hs_auth/config/role"
 	"github.com/unicsmcr/hs_auth/entities"
 	"github.com/unicsmcr/hs_auth/routers/api/models"
+	common2 "github.com/unicsmcr/hs_auth/routers/common"
```

Comment:
Do you think we could rename either this to `rcommon` or line 7 to be `v2 "github.com/unicsmcr/hs_auth/authorization/v2/common"`?